docs/bigcommerce-for-wordpress/setup/multi-site.md

```diff
@@ -65,27 +65,27 @@ Multiple sites can share the same API credentials or you can choose to create a
 
 3. For the OAuth Scopes, select the following default settings:
 
-**OAuth Scope**|**Default Selection**
-:-----:|:-----:
-Content|None
-Checkout Content|None
-Customers|Modify
-Customers Login|Login
-Information & Settings|Modify
-Marketing|Read-Only
-Orders|Read-Only
-Order Transactions|Read-Only
-Create Payments|None
-Get Payment Methods|Read-Only
-Products|Read-Only
-Themes|None
-Carts|Modify
-Checkouts|Modify
-Sites & Routes|Modify
-Channel Settings|Modify
-Channel Listings|Modify
-Storefront API Tokens|None
-Storefront API Customer Impersonation Tokens|None
+|               **OAuth Scope**                | **Default Selection** |
+|:--------------------------------------------:|:---------------------:|

```

Comment:
```suggestion
|:--------------------------------------------:|:----------------------|

```

It seems more readable for the list of OAuth scopes to be left-aligned.

Reply:
```suggestion
|:--------------------------------------------:|:----------------------|

```

The list of default selections could also be left-aligned, but I don't feel as strongly about this from a readability POV.

Reply:
Flushed them all left because, yes, it's way easier to read.